sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/sas/TableSasIpRange.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.sas;
+
+/**
+ * This type specifies a continuous range of IP addresses. It is used to limit permissions on SAS tokens. Null may be
+ * set if it is not desired to confine the sas permissions to an IP range.
+ */
+public final class TableSasIpRange {
```

Comment:
Any reason we aren't using `SasIpRange` from `azure-storage-common`?

Reply:
Please see my comment above.